### ISSUE TITLE
David's dead sprite in the Airseed Shrine

### ISSUE THREAD

**Issue by crumley467** [psywombats/mgne#493]
In the cutscene where David and BORG leave your party, David leaves first, then BORG talks about returning to the elder. During this conversation, whenever the text box disappears, you can see David standing on the stairway down, doing nothing.

**Comment by psywombats** [comment]
This will go away once we have 16x16 charas again

**Comment by crumley467** [comment]
no, its his whole sprite not just the top of his head. The top of the head thing i see in a couple places but i figured exactly what you said so i didn't open a ticket for it.
